### ISSUE TITLE
localization

### ISSUE THREAD

**Issue by kylemcdonald** [ofZach/pgElectron#19]
this seems like the perfect chance to separate our strings out :)

this is the kind of thing that might make sense to roll this ourselves instead of trying to use a library.

one way to do it would be to remove all the text from the HTML, and have a js object like this:

``` js
var strings = {
    "#platform": {
        "en": "Platform",
        "jp": "プラットフォーム"
    },
    ".addons": {
        "en": "Addons"
    },
};
```

where, if a language is not available it defaults to english. then in settings there would be a dropdown based on the available keys.

**Comment by Daandelange** [comment]
Ouch, little late to think about localisation :angel: 
Most text is contained in the `index.html` file... but some are baked into the `.js` files... so yes, "roll this ourselves" could be a better solution. :)

**Comment by kylemcdonald** [comment]
yes, didn't intend this to be something we release with 0.9.0, but i think once the design is basically in place we can break out the strings.
